.github/workflows/ci.yml

```diff
@@ -41,123 +36,122 @@ jobs:
         if: contains(matrix.os, 'ubuntu')
 
       - name: install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@v1
         with:
           toolchain: 1.68
           components: clippy
-          profile: minimal
-          override: true
 
       - name: restore cache
-        uses: Swatinem/rust-cache@v1
+        uses: Swatinem/rust-cache@v2
 
       # Clippy packages in deeper-to-higher dependency order
       - name: cargo clippy glazier
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --manifest-path=Cargo.toml --all-targets --features=x11 --no-default-features -- -D warnings
+        run: cargo clippy --manifest-path=Cargo.toml --all-targets --features=x11 --no-default-features -- -D warnings
```

Comment:
I think we're in one of these situations.
1) This comment and use of `--manifest-path` are out of date from the transition from druid
2) Or the comment doesn't explain the state well enough for me to understand this

If 1), we may as well tidy this up here

Reply:
I don't think the `--manifest-path=Cargo.toml` or comment are needed in any of the cases any longer. I thought about removing it as a separate commit in this PR and I guess I should have.

I will.